src/main/java/games/strategy/engine/framework/startup/mc/ServerModel.java

```diff
@@ -384,27 +384,15 @@ public void changeServerGameTo(final String gameName) {
     }
 
     @Override
-    public void changeToLatestAutosave(final SaveGameFileChooser.AUTOSAVE_TYPE typeOfAutosave) {
-      final HeadlessGameServer headless = HeadlessGameServer.getInstance();
-      if (headless == null) {
-        return;
-      }
-      final File save;
-      if (SaveGameFileChooser.AUTOSAVE_TYPE.AUTOSAVE.equals(typeOfAutosave)) {
-        save = new File(ClientContext.folderSettings().getSaveGamePath(), SaveGameFileChooser.getAutoSaveFileName());
-      } else if (SaveGameFileChooser.AUTOSAVE_TYPE.AUTOSAVE_ODD.equals(typeOfAutosave)) {
-        save = new File(ClientContext.folderSettings().getSaveGamePath(), SaveGameFileChooser.getAutoSaveOddFileName());
-      } else if (SaveGameFileChooser.AUTOSAVE_TYPE.AUTOSAVE_EVEN.equals(typeOfAutosave)) {
-        save =
-            new File(ClientContext.folderSettings().getSaveGamePath(), SaveGameFileChooser.getAutoSaveEvenFileName());
-      } else {
+    public void changeToLatestAutosave(final SaveGameFileChooser.AutoSaveType autoSaveType) {
```

Comment:
This was refactored after I noted that the only thing that changed was the second arg in the `File` constructor, for example ` SaveGameFileChooser.getAutoSaveFileName()`. Notably to refactor this those values/methods were moved to the `AutoSaveType` enum.